js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
+      this.firstWord = firstWord;
+    }
+    else {
+      this.firstWord = 'Fizz';
+    }
+
+    if (secondWord !== '') {
+      this.secondWord = secondWord;
+    }
+    else {
+      this.secondWord = 'Buzz';
+    }
+
+    console.log(this.firstWord);
```

Comment:
Try to not leave `console.log` statements behind; they're really useful for debugging, of course, but it is a little sloppy, and can actually cause performance issues (some browsers will slow down if repeating something like this out many times for every function call; think of a `console.log` from inside of a loop that's working through a large list of objects).